### PR TITLE
Add POST and PUT examples in namespaces API doc

### PR DESCRIPTION
**Pull request by hillaryfraley**
Add POST /namespaces and PUT /namespaces/:namespace examples.

Epic #1452

### CODE REVIEW

File: content/sensu-go/5.16/api/namespaces.md [hunk 1]
```diff
@@ -28,7 +28,8 @@ The `/namespaces` API endpoint provides HTTP GET access to [namespace][1] data.
 The following example demonstrates a request to the `/namespaces` API endpoint, resulting in a JSON array that contains [namespace definitions][1].
 
 {{< highlight shell >}}
-curl http://127.0.0.1:8080/api/core/v2/namespaces -H "Authorization: Bearer $SENSU_TOKEN"
+curl -X GET http://127.0.0.1:8080/api/core/v2/namespaces
```
Comment (Blue0ctober) [inline content/sensu-go/5.16/api/namespaces.md]: requires ' \ ' at the end of the line, probably should move 'curl -X GET' to it's own line for consistency

File: content/sensu-go/5.16/api/namespaces.md [hunk 2]
```diff
@@ -130,7 +165,8 @@ The `/user-namespaces` API endpoint provides HTTP GET access to the namespaces t
 The following example demonstrates a request to the `/user-namespaces` API endpoint, resulting in a JSON array that contains the namespaces the user has access to.
 
 {{< highlight shell >}}
-curl http://127.0.0.1:8080/api/enterprise/user-namespaces -H "Authorization: Bearer $SENSU_TOKEN"
+curl -X GET http://127.0.0.1:8080/api/enterprise/user-namespaces
```
Comment (Blue0ctober) [inline content/sensu-go/5.16/api/namespaces.md]: requires ' \ ' at the end of the line, probably should move 'curl -X GET' to it's own line for consistency

File: content/sensu-go/5.16/api/namespaces.md [hunk 3]
```diff
@@ -130,7 +165,8 @@ The `/user-namespaces` API endpoint provides HTTP GET access to the namespaces t
...
+-H "Authorization: Bearer $SENSU_TOKEN"
 [
```
Comment (Blue0ctober) [inline content/sensu-go/5.16/api/namespaces.md:173]: We should include either a new line and/or denote that everything after this is the example output and not the command that should be ran.

**Review by Blue0ctober — CHANGES_REQUESTED** [review]
See comments for recommended changes

File: content/sensu-go/5.16/api/namespaces.md (hunk 1 above)
Comment (hillaryfraley) [inline content/sensu-go/5.16/api/namespaces.md]:
```suggestion
curl -X GET \
http://127.0.0.1:8080/api/core/v2/namespaces \
```

File: content/sensu-go/5.16/api/namespaces.md (hunk 2 above)
Comment (hillaryfraley) [inline content/sensu-go/5.16/api/namespaces.md]:
```suggestion
curl -X GET \
http://127.0.0.1:8080/api/enterprise/user-namespaces \
```

File: content/sensu-go/5.16/api/namespaces.md [hunk 4]
```diff
@@ -130,7 +166,9 @@ The `/user-namespaces` API endpoint provides HTTP GET access to the namespaces t
 The following example demonstrates a request to the `/user-namespaces` API endpoint, resulting in a JSON array that contains the namespaces the user has access to.
 
 {{< highlight shell >}}
-curl http://127.0.0.1:8080/api/enterprise/user-namespaces -H "Authorization: Bearer $SENSU_TOKEN"
+curl -X GET \
+http://127.0.0.1:8080/api/enterprise/user-namespaces \
+-H "Authorization: Bearer $SENSU_TOKEN"
```
Comment (hillaryfraley) [inline content/sensu-go/5.16/api/namespaces.md]:
```suggestion
-H "Authorization: Bearer $SENSU_TOKEN"

```

File: content/sensu-go/5.16/api/namespaces.md (hunk 3 above)
Comment (hillaryfraley) [reply content/sensu-go/5.16/api/namespaces.md:173]: @Blue0ctober I added a blank line -- do you think it's ok like that? I looked at our other GET examples, and they generally include the response this way.

**Pull request merged**